jdaviz/container.vue

```diff
@@ -81,10 +81,6 @@
     z-index: 1;
     width: 24px;
   }
-  .imviz div.v-card.v-card--flat.v-sheet.v-sheet--tile {
-    /* black background beyond edges of canvas for canvas rotation */
-    background-color: black
-  }
```

Comment:
@bmorris3 - I vaguely remember this causing you troubles right?  Was there anything we did to workaround it that could/should be undone here as well?

Reply:
This isn't causing issues anymore, because we don't use canvas rotation. We now get a white background where there are no data from work starting in glue https://github.com/glue-viz/glue/pull/2482, which was first used in jdaviz in https://github.com/spacetelescope/jdaviz/pull/2671, and also used in the DQ layers. Removing this _should_ (I hope) have no effect.